services/blockchain-indexer/shared/dataService/business/pos/stakes.js

```diff
@@ -86,7 +86,7 @@ const getStakes = async params => {
 	stakesResponse.meta.staker = {
 		address: params.address,
 		name: accountInfo.name || null,
-		publicKey: accountInfo.publicKey || null,
+		publicKey: accountInfo.publicKey || params.publicKey || null,
```

Comment:
Could you also please ensure to asynchronously index the publicKey? We don't need to await the indexing.